libs/shared/otp/src/lib/otp.ts

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { timingSafeEqual, randomInt } from 'crypto';
+
+type OtpStorage = {
```

Comment:
Should this be an interface? 🤔

Reply:
What's the advantage of allowing people to augment this type?

Reply:
Sure, I can change it to an interface.

Typescript has a structural type system though so it won't stop anyone from passing an ad-hoc object.